tools/src/linter/SchemasValidator.ts

```diff
@@ -30,17 +26,19 @@ export default class SchemasValidator {
   root_folder: string
   spec: Record<string, any> = {}
   ajv: AJV
+  errors_parser: AjvErrorsParser
 
   constructor (root_folder: string, logger: Logger) {
     this.logger = logger
     this.root_folder = root_folder
     this.ajv = new AJV({ strict: true, discriminator: true })
     addFormats(this.ajv)
     for (const keyword of ADDITIONAL_KEYWORDS) this.ajv.addKeyword(keyword)
+    this.errors_parser = new AjvErrorsParser(this.ajv, {})
   }
 
   validate (): ValidationError[] {
-    this.spec = new OpenApiMerger(this.root_folder, this.logger).merge().components as Record<string, any>
+    this.spec = new OpenApiMerger(this.root_folder, new Logger(LogLevel.error)).merge().components as Record<string, any>
```

Comment:
Why would we not want to pass down the top level logger?

Reply:
Because we're printing out warnings for the mergers when linting, while those are not the concerns of the linter. Those warnings are already being printed out by running the merger itself. The goal was reducing noise for the linter.

Reply:
I see, makes sense.